src/jobrunner.rs

```diff
@@ -479,10 +479,10 @@ impl JobRunner {
     }
 
     /// If the user has specified an email address, send the contents of
-    fn sendemail(&self, email: &Option<String>, mut file: &File) {
-        if let Some(ref toaddr) = email {
+    fn sendemail(&self, job: &Job, mut file: &File) {
+        if let Some(ref toaddr) = job.rconf.email {
             let mut buf = Vec::new();
-            buf.extend(b"Subject: snare error\n\n");
+            buf.extend(format!("Subject: snare error when running {}\n\n", job.repo_id).as_bytes());
```

Comment:
Does it make sense to "run" a repo?

How about something like:
```
[snare] encountered errors for repo "wibble".
```

Reply:
Agreed that something else would be better. Try https://github.com/softdevteam/snare/pull/52/commits/feebdf968eb0a59a6401530aa982ec143c37ddd0.